modin/pandas/base.py

```diff
@@ -3455,36 +3455,50 @@ def tz_convert(self, tz, axis=0, level=None, copy=True):  # noqa: PR01, RT01, D2
         """
         Convert tz-aware axis to target time zone.
         """
-        axis = self._get_axis_number(axis)
-        if level is not None:
-            new_labels = (
-                pandas.Series(index=self.axes[axis]).tz_convert(tz, level=level).index
+        if hasattr(self._query_compiler, "tz_convert"):
```

Comment:
this doesn't feel right... I would propose to require this method but default-to-pandas in normal implementation.
This way - allowing different QCs to have or not to have a method - opens a can of worms IMHO I would rather not let be opened...

Reply:
Makes sense - let me wait on #6145 for this PR then, since we'll need the base's `_create_or_update_inplace` for these